components/widgets/inputs/users_emails_input.tsx

```diff
@@ -276,6 +276,12 @@ export default class UsersEmailsInput extends React.PureComponent<Props, State>
             if (this.props.onUsersLoad) {
                 this.props.onUsersLoad(options);
             }
+            const optionWithLabel = options.forEach((option) => {
+                option.label = option.username;
+            });
+            return new Promise((resolve) => {
```

Comment:
Why is this change needed? Setting the labels with `options.forEach....` is adequate. Its return value is undefined, so assigning it to `optionWithLabel` and returning the promise doesn't do something more than just setting the labels before setting state.